crates/matrix-sdk-common/src/timeout.rs

```diff
@@ -24,7 +24,7 @@ use tokio::time::timeout as tokio_timeout;
 
 /// Error type notifying that a timeout has elapsed.
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
-pub struct ElapsedError(());
+pub struct ElapsedError(pub ());
```

Comment:
If construction of this has become important, let's rather remove the field completely.

```suggestion
pub struct ElapsedError;
```

Reply:
sure, I was aiming to avoid breaking changes to existing code, but this makes sense